hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/export/ExportSpanConstants.java

```diff
@@ -0,0 +1,43 @@
+package org.hypertrace.core.graphql.span.export;
+
+import java.util.List;
+
+public interface ExportSpanConstants {
+  interface SpanAttributes {
+    String ID = "id";
+    String SERVICE_NAME = "serviceName";
+    String TRACE_ID = "traceId";
+    String PARENT_SPAN_ID = "parentSpanId";
+    String START_TIME = "startTime";
+    String END_TIME = "endTime";
+    String NAME = "displaySpanName";
+    String TAGS = "spanTags";
+
+    static List<String> getSpanAttributes() {
```

Comment:
nit: any reason for this to be a fn rather than a static const like the strings?

Reply:
will convert it to static list in follow-up PR.